### PR TITLE
more raf, transparent check

### PR DESCRIPTION
**Pull request by sun765**
The poster bug still exist even if we have already used two raf. I just add code to test whether the golden is semi transparent. 
image

### CODE REVIEW

File: packages/render-fidelity-tools/src/common.ts [hunk 1]
```diff
@@ -275,6 +281,11 @@ export class ImageComparator {
       throw new Error('Candidate image is colorless!');
     }
 
+    if (transparentPixelCount === imagePixelCount) {
```
Comment (sun765) [inline packages/render-fidelity-tools/src/common.ts:284]: If the candidate image is completely transparent, it's marked as colorless ,not transparent here.  Do you want it this way or do you want it to be marked as transparent?

File: packages/render-fidelity-tools/test/renderers/model-viewer/index.html [hunk 2]
```diff
@@ -60,7 +60,9 @@
         // the poster has completely faded away.
```
Comment (sun765) [inline packages/render-fidelity-tools/test/renderers/model-viewer/index.html:60]: Just added more raf to see if the bug still exist.

Comment (bejado) [reply packages/render-fidelity-tools/test/renderers/model-viewer/index.html:60]: Does this fix the bug? Maybe we can make the number of RAF configurable

File: packages/render-fidelity-tools/test/renderers/model-viewer/index.html [hunk 3]
```diff
@@ -60,7 +60,9 @@
         // the poster has completely faded away. 
         requestAnimationFrame(()=>{
           requestAnimationFrame(()=>{
-            this.dispatchEvent(new CustomEvent('model-ready'));
+            requestAnimationFrame(()=>{
+              this.dispatchEvent(new CustomEvent('model-ready'));
```
Comment (elalish) [inline packages/render-fidelity-tools/test/renderers/model-viewer/index.html:64]: Might be nice to add a comment about how it seems like three rafs are needed to get the poster to fade out reliably. Nice to document work-arounds.

**Pull request merged**